conda/activate.py

```diff
@@ -860,25 +892,37 @@ def native_path_to_unix(
     try:
         # if present, use cygpath to convert paths since its more reliable
         unix_path = run(
-            [cygpath, "--path", joined],
+            [cygpath, "--unix", "--path", joined],
             text=True,
             capture_output=True,
             check=True,
         ).stdout.strip()
     except FileNotFoundError:
+        log.warning(
+            "cygpath not available, falling back to less reliable path conversion"
+        )
+
         # fallback logic when cygpath is not available
         # i.e. conda without anything else installed
         def _translation(match):
             return "/" + (
-                match.group(1)
+                ((match.group("drive") or "").lower() + match.group("path"))
                 .replace("\\", "/")
                 .replace(":", "")
                 .replace("//", "/")
                 .rstrip("/")
             )
 
         unix_path = (
-            re.sub(r"([a-zA-Z]:[\/\\]+(?:[^:*?\"<>|;]+[\/\\]*)*)", _translation, joined)
+            re.sub(
+                r"""
+                (?P<drive>[a-z]:)?
+                (?P<path>[\/\\]+(?:[^:*?\"<>|;]+[\/\\]*)*)
+                """,
+                _translation,
+                joined,
+                flags=re.VERBOSE | re.IGNORECASE,
```

Comment:
Oh, huh, `re.IGNORECASE`, I thought Cygwin is case-sensitive, could this be an incompatibility perhaps?

Reply:
The `re.IGNORECASE` only impacts the drive letter matching here, updated to use `[A-Za-z]` instead.